imageroot/actions/configure-module/validate-input.json

```diff
@@ -8,7 +8,6 @@
             "api_user": "admin",
             "api_password": "admin",
             "host": "controller.nethserver.org",
-            "lets_encrypt": true,
```

Comment:
Why this has been removed?
`lets_encrypt` is still a required parameter

Reply:
it is a new behavior, lets_encrypt is not more a required parameter

```
lets_encrypt true # ask a LE certs and fails if not issued
lets_encrypt false # set the certs as obsolete and restart traefik
lets_encrypt missing # do nothing
```